examples/stripe-script/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "stripe-script",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "toolpad dev",
+    "build": "toolpad build",
+    "start": "toolpad start"
+  },
+  "dependencies": {
+    "@mui/toolpad": "^0.1.22",
+    "child_process": "^1.0.2",
+    "fs": "^0.0.1-security",
```

Comment:
Are these not included with node? (`fs`, `path`, `child_process`...)
Do we need to include them in package.json?